### DISCUSSION TITLE
视觉问题

### DISCUSSION THREAD

**Discussion by robin12jbj** [timqian/openprompt.co#46] [General]
长一些的回答内容，页面一直在刷新，观感不好，想要从第一句看起，发现一直在滚动

**Comment by timqian** [comment]
这里为了帮用户省去下滑的操作，有新内容时会自动滚到最下面。
如果不自动往下滚，用户每次输入完成就需要自己往下翻一下，可能会有不变。

不过自动往下滚动确实也有你说的这个问题，特别是在手机上使用的时候。需要考虑下怎么做比较好
